### PR TITLE
ODF4.17 UI deployment, add 'data protection' page

### PR DESCRIPTION
**Pull request by OdedViner**

### CODE REVIEW

**Comment by OdedViner** [comment]
Deployment via UI pass: https://url.corp.redhat.com/e4f15d0 https://url.corp.redhat.com/6d80eec
@petr-balogh @dahorak Can we merge it?

**Review by petr-balogh — APPROVED** [review]

**Comment by petr-balogh** [comment]
Once we get one more approval, we can ;)

**Review by prsurve — APPROVED** [review]

**Review by vavuthu — APPROVED** [review]

**Review by DanielOsypenko — APPROVED** [review]

**Pull request merged**
